rails/config/locales/en/administrate.en.yml

```diff
@@ -34,6 +34,7 @@ en:
         mapbox_style_url: Mapbox style URL (for online maps)
         mapbox_access_token: Mapbox access token associated with the style
         mapbox_3d: Activate 3D Terrain view for online map
+        map_projection: Set projection for map
```

Comment:
Let's populate this translation in `models.en.yml` as well. 

We should also probably add the key to the other language files as well even if the translated value is currently English.